packages/lesswrong/components/ultraFeed/UltraFeedThreadCommentsList.tsx

```diff
@@ -164,7 +166,7 @@ const UltraFeedThreadCommentsList = ({
                 }}
                 onPostTitleClick={onPostExpansion}
                 onChangeDisplayStatus={(newStatus) => onSetDisplayStatus(cId, newStatus)}
-                showPostTitle={isFirstItem && !postInitiallyExpanded}
+                showPostTitle={focusedCommentId ? cId === focusedCommentId : (isFirstItem && !postInitiallyExpanded)}
```

Comment:
nit: complicated enough expression that I'd rather have it extracted to its own line, at least